### ISSUE TITLE
Accessibility: Missing Null Alt on Profile Images

### ISSUE THREAD

**Issue by michaelspellacy** [voxpelli/webpage-webmentions#125]
Hi, 

Can you please add a null alt to the profile images being generated by your script? Not having it is an accessibility issue and can cause the img src to be read back assistive technology, which is not desirable. Thank you so much! 

https://www.screencast.com/t/ocmBRzL4V
